Kagorora/Domestic-workers-system
# Setting up the project environment

#### What does this PR do?
- add dependencies and folder structure template
#### Description of Task to be completed?
- configure eslint
- configure babel
- install express
- install postgres and sequelize
- install bcrypt
- install mocha and chai
- install jasonWebToken
- install uuid
#### How should this be manually tested?
- Clone the repo
- run `npm i`
- check `package.json`

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/170876439
